ironfish-cli/src/command.ts

```diff
@@ -109,6 +112,7 @@ export abstract class IronfishCommand extends Command {
     const configFlag = getFlag(flags, ConfigFlagKey)
 
     const configOverrides: Partial<ConfigOptions> = {}
+    const internalConfigOverrides: Partial<InternalOptions> = {}
```

Comment:
This says config, but config is separate from internal.

```suggestion
    const internalOverrides: Partial<InternalOptions> = {}
```